features/svg.yml

```diff
@@ -428,7 +428,6 @@ compat_features:
   - svg.elements.script
   - svg.elements.script.href
   - svg.elements.script.type
-  - svg.elements.script.xlink_href
```

Comment:
Will be marked as deprecated in https://github.com/mdn/browser-compat-data/pull/25477, but that doesn't block this change.